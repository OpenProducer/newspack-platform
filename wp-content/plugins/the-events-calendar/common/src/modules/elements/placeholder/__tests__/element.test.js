import React from 'react';
import Placeholder from '../element';

describe( '<Placeholder> component', () => {
	test( 'Default behavior', () => {
		const component = renderer.create(
			<Placeholder>Custom Text</Placeholder>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	test( 'Custom Class name attached', () => {
		const component = renderer.create(
			<Placeholder className='custom-class-name'>Custom Text</Placeholder>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
