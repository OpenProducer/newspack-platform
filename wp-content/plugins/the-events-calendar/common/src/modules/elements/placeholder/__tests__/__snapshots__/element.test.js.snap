// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Placeholder> component Custom Class name attached 1`] = `
<div
  className="tribe-editor__placeholder custom-class-name"
>
  Custom Text
</div>
`;

exports[`<Placeholder> component Default behavior 1`] = `
<div
  className="tribe-editor__placeholder"
>
  Custom Text
</div>
`;
